Frends.CSV.Create/Frends.CSV.Create/Frends.CSV.Create.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
 	<TargetFrameworks>net6.0</TargetFrameworks>
-	<Version>1.3.0</Version>
+	<Version>1.3.1</Version>
```

Comment:
The version bump should be to 1.4.0 instead of 1.3.1 (like we discussed today, the final digit is for testing and development) 😄